daffodil-test/src/test/resources/org/apache/daffodil/section06/namespaces/namespaces.tdml

```diff
@@ -167,8 +167,9 @@
   <tdml:parserTestCase name="ibm_format_compat_02"
     root="ABC" model="ABC_IBM.dfdl.xsd" description=""
     roundTrip="twoPass">
-    <tdml:document><![CDATA[a
-]]></tdml:document>
+    <tdml:document>
+	  <tdml:documentPart type="text" replaceDFDLEntities="true"><![CDATA[a%LF;]]></tdml:documentPart>
```

Comment:
I'm not going to suggest changing these, but in these short-line cases, This would also have worked since the character being inserted is a legal XML character:
<document>a&#x0A;</document>
This has little chance of being line-wrapped. The drawback, is if the character being inserted is not a legal char like LF, but is something like NUL, then this won't work, and the longer syntax as used in the code, with the DFDL entities being used, that would be the only way to express it.

Reply:
Did something in your comment get stripped out by github? I'm guessing you mean to say  ``a&#xA;`` and github replaced the xA it with an actual linefeed? Using backticks seems to escape it and make it show up.

Reply:
I edited to fix the above so it is readable. Thanks for the catch.